orange/src/main/java/com/hartwig/hmftools/orange/report/chapters/FrontPageChapter.java

```diff
@@ -66,46 +67,58 @@ public class FrontPageChapter implements ReportChapter {
     private final ReportResources reportResources;
 
     public FrontPageChapter(@NotNull final OrangeRecord report, @NotNull final PlotPathResolver plotPathResolver,
-            @NotNull final ReportResources reportResources) {
+            @NotNull final ReportResources reportResources)
+    {
         this.report = report;
         this.plotPathResolver = plotPathResolver;
         this.reportResources = reportResources;
     }
 
     @NotNull
     @Override
-    public String name() {
+    public String name()
+    {
         return "Front Page";
     }
 
     @NotNull
     @Override
-    public PageSize pageSize() {
+    public PageSize pageSize()
+    {
         return PageSize.A4;
     }
 
     @Override
-    public void render(@NotNull Document document) {
+    public void render(@NotNull Document document)
+    {
         addSummaryTable(document);
         addDetailsAndPlots(document);
     }
 
-    private void addSummaryTable(@NotNull Document document) {
+    private void addSummaryTable(@NotNull Document document)
+    {
         Cells cells = new Cells(reportResources);
-        Table table = Tables.createContent(contentWidth(),
-                new float[] { 3, 2, 1 },
-                new Cell[] { cells.createHeader("Configured Primary Tumor"), cells.createHeader("Cuppa Cancer Type"),
-                        cells.createHeader("QC") });
+
+        float[] headerComponents = new float[] { 3, 2, 1 };
+        Cell[] headerCells = headerCells = new Cell[] {
+                cells.createHeader("Configured Primary Tumor"),
+                !report.tumorOnlyMode() ? cells.createHeader("Cuppa Cancer Type") : cells.createHeader("Tumor-Only"),
+                cells.createHeader("QC") };
+
+        Table table = Tables.createContent(contentWidth(), headerComponents, headerCells);
 
         table.addCell(cells.createContent(configuredPrimaryTumor(report.configuredPrimaryTumor())));
-        table.addCell(cells.createContent(cuppaCancerType(report.cuppa())));
+        table.addCell(cells.createContent(!report.tumorOnlyMode() ? cuppaCancerType(report.cuppa()) : ""));
```

Comment:
I think it is clear as-is since there is no reference to Cuppa in this mode, and it indicates the mode for the run. A more explict run-mode could instead be added to the front page, but we would still leave out any reference to Cuppa (as per Chord, Sigs etc).

Reply:
I quite like the idea of mentioning some information about the mode somewhere up top. In ACTIN it's confusing when the CUPPA results are displayed on front page whether ORANGE was run on WGS or WGTS. In case of WGTS we display the DNA+RNA combined score rather than the DNA-combined score. This only becomes clear when scrolling down the report. 

Maybe the way its proposed here is not ideal, could also just remove the entire CUPPA column in case of no-CUPPA-data but also fine with the current proposal tbh